crates/wallet/src/wallet/mod.rs

```diff
@@ -2463,9 +2455,26 @@ impl Wallet {
     ///
     /// This operation is generally only used when importing or restoring a previously used wallet
     /// in which the list of used scripts is not known.
+    ///
+    /// Use [`start_full_scan_at`](Wallet::start_full_scan_at) to manually specify the `start_time`
+    /// of the full scan or if `feature = "std"` is not available.
+    #[cfg(feature = "std")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "std")))]
     pub fn start_full_scan(&self) -> FullScanRequestBuilder<KeychainKind> {
+        let start_time = std::time::UNIX_EPOCH
+            .elapsed()
+            .expect("time must no go backwards")
+            .as_secs();
+        self.start_full_scan_at(start_time)
+    }
+
+    /// Create a [`FullScanRequest] for this wallet.
```

Comment:
Looks like this was pre-existing.

```suggestion
    /// Create a [`FullScanRequest`] for this wallet.
```